test/widgets/conftest.py

```diff
@@ -74,3 +77,39 @@ class MinimalConf(libqtile.confreader.Config):
         screens = []
 
     return MinimalConf
+
+
+@pytest.fixture(scope='function')
+def dbus(monkeypatch):
+    # dbus-launch is provided by "dbus-x11" package
```

Comment:
*on Ubuntu. On Arch it's provided by `dbus` which fortunately has no X dependencies.

Reply:
Indeed. This was more to explain why the dbus-x11 package was added to CI.

Reply:
Yeah good to keep that